generateMetadataJsonFromResources.js

```diff
@@ -0,0 +1,20 @@
+const fs = require("fs");
+const path = require("path");
+
+const metadataJsonFile = "metadata.json";
+const resourcesDir = "resources";
+
+console.debug(`Creating ${metadataJsonFile}...`);
+const resources = fs.readdirSync(resourcesDir);
+const metadataList = resources.map((file) => {
+  return {
+    name: path.basename(file, ".js"),
+    aliases: [path.basename(file, ".js")],
+    kind: { mime: "application/javascript" },
+    resourcePath: path.basename(file),
```

Comment:
It's totally fine for `aliases` to be an empty list; there's no benefit to adding the same exact name twice.

also, just noting that we might not always be dealing with _only_ JS resources, but I suppose that can be dealt with in the future if it happens

Reply:
Changed the script name to reflect that it's only valid for JS resources, and made aliases be an empty list.